asv_bench/benchmarks/frame_methods.py

```diff
@@ -512,3 +512,22 @@ def time_nlargest(self, keep):
 
     def time_nsmallest(self, keep):
         self.df.nsmallest(100, 'A', keep=keep)
+
+
+class Describe(object):
+
+    goal_time = 0.2
+
+    def setup(self):
+        np.random.seed(123)
```

Comment:
You can remove the random seed; this is handled when `setup` is imported at the top (`from .pandas_vb_common import setup`)